examples/yum_collection_migration_spec.yml

```diff
@@ -0,0 +1,40 @@
+# a draft of a spec file to migrate content from core to a collection, for test purposes
+
+yum_collection:
+  action:
+    yum.py
+  module_utils:
+    yumdnf.py
+  modules:
+    packaging/os/yum.py
+  units:
+    test_yum.py
+  targets:
+    yum
+
+  # adding more to test the rest of plugin types
+  # missing networking ones for now
+  become:
+    sudo.py
+  cache:
+    memcached.py
+  callback:
+    default.py
+  connection:
+    local.py
+  docs_fragments:
+    shell_common.py
+  filter:
+    core.py
+  inventory:
+    ini.py
+  lookup:
+    env.py
+  test:
+    core.py
+  shell:
+    sh.py
+  strategy:
+    free.py
+    linear.py
```

Comment:
```suggestion
   - linear.py
```

Reply:
```suggestion
    linear.py:
```

Reply:
we should make them all a list then, not just the last one

Reply:
sure, that was just an example of what I meant.